notebooks/cox2.ipynb

```diff
@@ -23,7 +23,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 6,
```

Comment:
Tiny and not a thing for this notebook but I've been trying to rerun notebooks before upload to ensure they execute start to finish and pass all appropriate tests.

Reply:
i thought I'd done this